spyder_notebook/notebookplugin.py

```diff
@@ -33,7 +33,6 @@
 from .widgets.client import NotebookClient
 
 
-NOTEBOOK_TMPDIR = tempfile.gettempdir()

```

Comment:
You still need to preserve this variable, only now it becomes

    NOTEBOOK_TMPDIR = osp.join(TEMPDIR, 'notebooks')